### DISCUSSION TITLE
possible accounts with no transfers

### DISCUSSION THREAD

**Discussion by jester223** [DSJAS/DSJAS#36] [Ideas]
I thought it would a cool idea if we could set certain bank account to have no outbound transfers. It would let us to be able sell certain accounts types like market or roth ira accounts. Basically i want make something like this as way to tease scammers  of here some money you can't touch. it could also sell the bank as being more legit i think.

**Comment by SinisterSpatula** [comment]
That's a good idea. 👍

**Comment by ejv2** [comment]
This was planned with the "disabled" accounts feature, but I never properly programmed it in. Should be simple enough to add. I'll put it on the todo list.

**Reply by ejv2** [reply]
Experimental support added in b486f61bf

**Reply by ejv2** [reply]
I'm going to mark this as complete, as its been in the site for some time now and I've not recieved any further bug reports. Thanks for the suggestion!

**Comment by ejv2** [comment]
Minor update to this. Until 84c59c0, it was still possible to make outbound transfers if the source account was disabled so long as the destination account was not. I have now fixed this and both accounts are now checked.

**Reply by ejv2** [reply]
Closing as complete.
